syncthing.te

```diff
@@ -0,0 +1,45 @@
+policy_module(syncthing, 1.0.0)
+
+# define types
+type syncthing_t;
+type syncthing_exec_t;
+userdom_user_application_domain(syncthing_t, syncthing_exec_t)
+
+type syncthing_config_home_t;
+userdom_user_home_content(syncthing_config_home_t)
+
+# file permissions
+fs_getattr_xattr_fs(syncthing_t)
+
+userdom_manage_user_home_content(syncthing_t)
```

Comment:
> That means that you would also need to add type transition rules so that files and dirs created in ~ will be created with an auto type transition from user_home_dir_t to user_home_t

How should I accomplish that? There are quite a few different folders/types which could be created by `syncthing_t` on start, could be `audio_home_t`, etc.